apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -209,6 +209,11 @@
                     "Maximum time (milliseconds) for each attempt to mine a block with a specific nonce.",
                     "type" : "integer"
                 },
+                "expected_mine_rate" : {
+                    "description" : "Expected mine rate (milliseconds) between blocks. Used in governance.",
+                    "type" : "integer",
+                    "default": 300000
```

Comment:
Minimum strictly greater then 0 beneficial though not mandatory.

Reply:
just pushed 937da894